### PR TITLE
feat: add fancy_text command

### PR DESCRIPTION
**Pull request by m4tx**

### CODE REVIEW

File: chombot-kcc/src/chombot.rs
Comment (seqre) [inline chombot-kcc/src/chombot.rs:1]: That should be a separate PR.

**Review by Iipin — APPROVED** [review]
outstanding, keep it up

File: chombot-kcc/src/slash_commands/fancy_text.rs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            ("Text", &text),
+        ])
+        .send()
+        .await
+        .map_err(|err| FancyTextFetchError::FetchError(err.into()))?
+        .json()
+        .await
+        .map_err(|err| FancyTextFetchError::ParseError(err.into()))?;
+
+    let image = create_chombot_http_client_insecure()
```
Comment (seqre) [inline chombot-kcc/src/slash_commands/fancy_text.rs:73]: This is unacceptable; there shouldn't be any case of ignoring certificate validation for connections over the public Internet.

**Review by Iipin — APPROVED** [review]

**Review by seqre — CHANGES_REQUESTED** [review]
Remove insecure get

File: chombot-kcc/src/slash_commands/fancy_text.rs (hunk 1 above)
Comment (seqre) [reply chombot-kcc/src/slash_commands/fancy_text.rs:73]: It doesn't matter. You need to use different upstream of figure this out.

Comment (seqre) [reply chombot-kcc/src/slash_commands/fancy_text.rs:73]: We should strive to maintain high security standards and not push code like that.

Comment (m4tx) [reply chombot-kcc/src/slash_commands/fancy_text.rs:73]: Sadly, it seems the upstream server is the root of the issue, as I can see the same certificate error across different operating systems and tooling. Also, I don't really see much of a danger here, since this is a static image anyway, and there's no private info being sent here, so TLS doesn't add much of a seqre-ity here.

Comment (Iipin) [reply chombot-kcc/src/slash_commands/fancy_text.rs:73]: lmao
